### ISSUE TITLE
Date conflict

### ISSUE THREAD

**Issue by ashdavies** [BoD/klibnotion#27]
Importing Date from `org.jraf.klibnotion.model.date` can cause conflicts with DateTime libs or locally defined libraries, especially when defined with aliases, we should try to coalesce with the kotlinx and java.ltime library, with respect to multiplatform

**Comment by BoD** [comment]
Yeah that was a poor decision, I should have named it `NotionDate` or something, to avoid conflicts. I could rename it but keep a typealias to avoid breaking changes (but marked `@Deprecated`).
